app/services/incoming_text_message_service.rb

```diff
@@ -22,31 +22,16 @@ def self.process(params)
     event_name = client_count > 1 ? "client_found_multiple" : "client_found"
     DatadogApi.increment("twilio.incoming_text_messages.#{event_name}")
 
-    # process attachments once
-    attachments = TwilioService.new.parse_attachments(params)
-
     clients.map do |client|
-      documents = attachments.map do |attachment|
-        Document.new(
-          client: client,
-          document_type: DocumentTypes::TextMessageAttachment.key,
-          upload: {
-            io: StringIO.new(attachment[:body]),
-            filename: attachment[:filename],
-            content_type: attachment[:content_type],
-            identify: false
-          }
-        )
-      end
-
       contact_record = IncomingTextMessage.create!(
         body: params["Body"],
         received_at: DateTime.now,
         from_phone_number: phone_number,
         client: client,
-        documents: documents
```

Comment:
what did this do, and is it ok that we're losing it?

Reply:
I think we're doing:
```
    IncomingTextMessage.find(incoming_text_message_id).update(documents: documents)
```
in the job instead of setting it on create